src/test/java/fr/insee/pearljam/api/campaign/dto/output/CommunicationTemplateResponseDtoTest.java

```diff
@@ -26,13 +26,11 @@ void testFromModel() {
         // Then
         assertThat(communicationDtos).hasSize(2);
 
-        CommunicationTemplateResponseDto dto1 = communicationDtos.get(0);
-        assertThat(dto1.id()).isEqualTo(1L);
+        CommunicationTemplateResponseDto dto1 = communicationDtos.getFirst();
```

Comment:
need to keep foreignKey test

  assertThat(dto1.id()).isEqualTo(1L);
  => assert on campaignId and meshuggahId